etc/kayobe/environments/ci-multinode/README.md

```diff
@@ -1,48 +1,293 @@
+      --add-repo \
+      https://download.docker.com/linux/centos/docker-ce.repo
+  sudo dnf install docker-ce
+else
+  sudo apt update
+  sudo apt install -y docker.io
+fi
+```
+
+2. Start the Docker service
+
+```
+sudo systemctl start docker
+```
+
+3. Inside the kayobe-config root directory initialise the submodules
+
+```
+git submodule init
+git submodule update
+```
+
+4. Build Docker image
+
+```
+sudo DOCKER_BUILDKIT=1 docker build --file .automation/docker/kayobe/Dockerfile --tag kayobe:latest .
+```
+
+### Configure Test Resources
+
+> ⚠️ **WIP** configuration and instructions to follow soon. ⚠️
+
+In some instances the `configure-aio-resources.yml` may provide you with enough to work with and pass tempest tests. This can be achieved with the following;
+
+```
+kayobe playbook run etc/kayobe/ansible/configure-aio-resources.yml
+```
+
+```
+cp .automation.conf/tempest/tempest-{ci-aio,$KAYOBE_ENVIRONMENT}.overrides.conf
+```
+
+When a more comprehensive openstack setup is required you can use [Openstack Config Multinode](https://github.com/stackhpc/openstack-config-multinode) 
+
+### Running Tempest Tests
+
+1. Make a directory to store the tempest outputs
+
+```
+mkdir -p tempest-artifacts
+```
+
+2. Ensure the private key for kayobe has been set
+
+```
+export KAYOBE_AUTOMATION_SSH_PRIVATE_KEY=$(cat ~/.ssh/id_rsa)
+```
+
+3. Update your tempest inventory file with your seed hostname
+
+```
+vi ~/src/stackhpc-kayobe-config/etc/kayobe/environments/ci-multinode/inventory/kayobe-automation
+```
+
+4. Run the tempest test suite
+
+```
+sudo -E docker run -it --rm --network host -v $(pwd):/stack/kayobe-automation-env/src/kayobe-config -v $(pwd)/tempest-artifacts:/stack/tempest-artifacts -e KAYOBE_ENVIRONMENT -e KAYOBE_VAULT_PASSWORD -e KAYOBE_AUTOMATION_SSH_PRIVATE_KEY kayobe:latest /stack/kayobe-automation-env/src/kayobe-config/.automation/pipeline/tempest.sh -e ansible_user=stack
+```
+
+Once the test suite has finished you can view the contents of `${KAYOBE_CONFIG_PATH}/tempest-artifacts/failed_tests` which should be empty. You may also download a copy of `rally-verify-report.html` to review allowing you to ensure all expected tests were carried out. `scp centos@{{ ANSIBLE_HOST_IP }}:/home/centos/src/kayobe-config/tempest-artifacts/rally-verify-report.html ~/Downloads/rally-verify-report.html`
+
+## Tips & Tricks
+
+This section will be used to list useful commands and software that can be used to investigate and debug the mutltinode environment
+
+- When running the Tempest tests you may want to view the current progress which can be achieved by running `sudo docker logs --follow ${rally_docker_container_name}`
+
+- If need to access the network from your local machine or Ansible Control Host you can use `sshuttle` which operates as a transparent proxy over SSH. For example to access Horizon on the public network you can `sshuttle -r centos@${controller_ip} ${public_network_cidr}` This can also be used with `openstackclient`.
```

Comment:
Let's make a new section before the tempest one about accessing and using the cloud.

Also, we know the public CIDR so we could include that in the command. For SMS lab is there a bit more to it than this? The Ansible control host typically won't have a public IP